### ISSUE TITLE
Tune rule descriptions to be more specific

### ISSUE THREAD

**Issue by amesga** [daisy/ace-core#41]
Some rules have multiple possible infractions listed in the description. For example, image-alt has six possible reasons why the image was flagged.

- Element does not have an alt attribute
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible
- Element has no title attribute or the title attribute is empty
- Element's default semantics were not overridden with role="presentation"
- Element's default semantics were not overridden with role="none"

Having the Ace tool detect the specific cause of the error and display only that in the description, rather than the 6 possible causes, would go a long way towards being able to hand the Ace report off to a client as their instructions to fix their file.

**Comment by amesga** [comment]
Ah I see. I was interpreting the list of options as Ace not knowing what specifically triggered the error. So you're saying any of the options, if applicable, will fix the error? I will go back and edit some pages with this in mind.

**Comment by rdeltour** [comment]
> So you're saying any of the options, if applicable, will fix the error?

Correct.

I'll close the issue as invalid, feel free to reopen or create a new one if other descriptions are problematic!
Note however that most of them come from the underlying Web accessibility checker, so we may not have direct control over these.
